addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -3,11 +3,12 @@
 
 from odoo import Command
 from odoo.addons.event_booth_sale.tests.common import TestEventBoothSaleCommon
-from odoo.tests.common import users
+from odoo.addons.sale.tests.common import TestSaleCommon
+from odoo.tests.common import tagged, users
 from odoo.tools import float_compare
 
-
-class TestEventBoothSale(TestEventBoothSaleCommon):
+@tagged('post_install', '-at_install')
+class TestEventBoothSale(TestEventBoothSaleCommon, TestSaleCommon):
```

Comment:
Why changing this ? I don't see big changes in the test requiring it ?

Reply:
@tde-banana-odoo @flch-odoo 
Sorry for the delay. It's indeed because we need to have the COA already set. I updated the test by separating them, is it more sufficient?

Reply:
Hi @nboulif 
Thanks for the follow-up! Wile it works like this, there is a lot of duplicated code. So,
* Would it work instead to run the whole test flow post install in one class? That would be fine too. (But technically, there is something tricky about inherting from TestEventBoothSaleCommon and TestSaleCommon.
* Otherwise, maybe you could you make the second class inherit from the first (so `TestEventBoothSale` instead of `TestEventBoothSaleCommon` and override the test with the new one?